go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/runatlantis/atlantis
 go 1.22
 
 require (
+	code.gitea.io/sdk/gitea v0.17.1
 	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/alicebob/miniredis/v2 v2.31.1
 	github.com/bradleyfalzon/ghinstallation/v2 v2.9.0
 	github.com/briandowns/spinner v1.23.0
 	github.com/cactus/go-statsd-client/v5 v5.1.0
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
-	github.com/go-playground/validator/v10 v10.18.0
+	github.com/go-playground/validator/v10 v10.17.0
```

Comment:
Why are dependencies getting downgraded?

Reply:
That's a good question. I'll look into that.